test/src/cli/testCliRunner.js

```diff
@@ -32,8 +32,24 @@ describe('Test CLI Runner', function() {
       }
     };
 
+    let promiseConfig = new Promise(resolve => {
+      let config = {
+        src_folders: ['promiseTests'],
+        test_settings: {
+          default: {
+            silent: true
+          }
+        }
+      };
+
+      setTimeout(() => {
+        resolve(config);
+      }, 1000);
```

Comment:
1000ms is an unnecessarily high delay to add to the test. This can be simplified as:

```js
const promiseConfig = Promise.resolve({
  src_folders: ['promiseTests'],
  test_settings: {
    default: {
      silent: true
     }
  }
});
```